lib/game_runner.rb

```diff
@@ -49,11 +55,16 @@ def print_grid
     grid_printer.print_grid
   end
 
+  def mark_grid_with_player_input
+    grid.mark(get_valid_player_coordinate, 'X')
+  end
+
+  def mark_grid_with_computer_input
+    grid.mark(grid.get_computer_choice, 'O')
```

Comment:
This is kind of odd. You are making the grid and also asking the grid for the computer's choice. Seems like either the grid shouldn't know how to get a computer player's choice. But the real smell is passing something into grid that it already knows.

Reply:
Right, I thought something was amiss here. Moving the 'X' and 'O' to the grid simplified this!